purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -3634,220 +3634,6 @@ class PurchasesTest {
         }
     }
 
-    @Test
```

Comment:
I'm guess yes, but just to double-check: do we have equivalent tests for the non-deprecated API?

Reply:
Yes! Lines 3852-3880 😄